server/sched.go

```diff
@@ -43,10 +43,13 @@ type Scheduler struct {
 	getGpuFn    func() gpu.GpuInfoList
 }
 
-// TODO set this to zero after a release or two, to enable multiple models by default
-var loadedMax = 1          // Maximum runners; < 1 maps to as many as will fit in VRAM (unlimited for CPU runners)
-var maxQueuedRequests = 10 // TODO configurable
-var numParallel = 1
+var (
+	// TODO set this to zero after a release or two, to enable multiple models by default
+	loadedMax         = 1 // Maximum runners; < 1 maps to as many as will fit in VRAM (unlimited for CPU runners)
+	maxQueuedRequests = 50
```

Comment:
`1000` or even `10000` seems like a more reasonable default to avoid scenarios where there are many queued requests in error

We really, really don't want anyone hitting this accidentally in normal usage

Reply:
I've bumepd to 512.  Since this allocates multiple buffered channels and has to store those pending requests, I wanted to be careful we don't set this too large and wind up creating a DOS scenario where all those pending requests gobble up so many resources the system can't process the requests in flight.

Reply:
that seems like a great balance!